### PR TITLE
Fix fibery import for documents by public id

### PR DESCRIPTION
**Pull request by UladBohdan**

### CODE REVIEW

**Review by bantamer — DISMISSED** [review]

File: src/services/fibery/index.js [hunk 1]
```diff
@@ -316,7 +316,24 @@ const getTinderGame = async ({ fiberyPublicId }) => {
 };
 
 const getDocument = async fiberyPublicID => {
-  const [document] = await fibery.entity.query(DOCUMENT_VIEW, { $id: fiberyPublicID });
+  const response = await fetch('https://wir.fibery.io/api/views/json-rpc', {
```
Comment (Drapegnik) [inline src/services/fibery/index.js:319]: Maybe add simple wrapper around `fetch`?

Something like simplified version of [this](https://github.com/babajka/babajka-frontend/blob/7b4670d68615497386c5a849f56126376bedec14/utils/request.js#L21)

**Review by Drapegnik — DISMISSED** [review]

File: src/services/fibery/index.js (hunk 1 above)
Comment (UladBohdan) [reply src/services/fibery/index.js:319]: Yeah, I probably should

File: src/services/fibery/index.js [hunk 2]
```diff
@@ -316,7 +316,27 @@ const getTinderGame = async ({ fiberyPublicId }) => {
...
+    params: {
+      filter: {
+        publicIds: [fiberyPublicID],
+      },
+    },
+  };
+  const headers = {
+    Authorization: `Bearer ${config.services.fibery.token}`,
+  };
+  const response = await makeExternalRequest(
```
Comment (Drapegnik) [inline src/services/fibery/index.js:333]:
```suggestion
  const { result: [document] } = await makeExternalRequest(
```

Comment (UladBohdan) [reply src/services/fibery/index.js]: Done.

**Review by Drapegnik — APPROVED** [review]

**Pull request merged**